pcsx2/Sifcmd.h

```diff
@@ -16,178 +16,12 @@
 #ifndef __SIFCMD_H__
 #define __SIFCMD_H__
 
-/* from sifcmd.h */
```

Comment:
Would it worth keeping these around for documentation purposes? Or perhaps moving them into the documentation directory.

Reply:
Looks like all that information can be found here https://github.com/ps2dev/ps2sdk/blob/master/ee/kernel/include/sifrpc.h

So probably not a lot of reason to keep it hanging around in the code.